lib/atomic_web/components/calendar/calendar.ex

```diff
@@ -85,14 +86,23 @@ defmodule AtomicWeb.Components.Calendar do
               </div>
               <div class="hidden md:ml-4 md:flex md:items-center">
                 <div class="relative">
-                  <a @click="mode_view = !mode_view" class="flex cursor-pointer items-center py-2 pr-2 pl-3 text-sm font-medium text-zinc-700 hover:bg-zinc-50" id="menu-button" aria-expanded="false" aria-haspopup="true">
-                    <%= if @mode == "month" do
-                      gettext("Month view")
-                    else
-                      gettext("Week view")
-                    end %>
-                    <.icon name={:chevron_down} solid class="ml-2 h-5 w-5 text-zinc-400" />
-                  </a>
+                  <.dropdown
+                    orientation={:down}
+                    id="calendar-dropdown"
+                    items={[
+                      %{name: gettext("Week view"), link: @current_week_path},
+                      %{name: gettext("Month view"), link: @current_month_path}
+                    ]}
+                  >
+                    <.button color={:white}>
+                      <%= if @mode == "month" do %>
+                        <%= gettext("Month view") %>
+                      <% else %>
+                        <%= gettext("Week view") %>
+                      <% end %>
+                      <.icon name={:chevron_down} solid class="-mr-1 ml-2 h-5 w-5" />
```

Comment:
btw, you can actually add the icon via the button component thought the following attributes:
```suggestion
                    <.button color={:white} icon={:chevron_down} icon_position={:right} icon_variant={:solid}>
                      <%= if @mode == "month" do %>
                        <%= gettext("Month view") %>
                      <% else %>
                        <%= gettext("Week view") %>
                      <% end %>
```